packages/core/content-manager/server/src/history/index.ts

```diff
@@ -20,6 +20,9 @@ const getFeature = (): Partial<Plugin.LoadedPlugin> => {
         // Start recording history and saving history versions
         getService(strapi, 'history').init();
       },
+      destroy({ strapi }) {
+        getService(strapi, 'history').destroy();
```

Comment:
you could store the result of getService(strapi, 'history') in a historyService variable, since it's called twice

Reply:
I'm not sure I can since we need the the strapi argument to be available 🤔 . Unless you see another way...

Reply:
ah yes you're right